node/src/components/Navbar.js

```diff
@@ -24,10 +12,13 @@ class Navbar extends JSXComponent {
 					<a href="/about">About</a>
 				</li>
 				<li>
-					<a onClick={this.signOut}>Sign Out</a>
-				</li>
-				<li>
-					<span class="username"></span>
+					{Auth.currentUser ?
+						<span>
+							{Auth.currentUser.name}
+							<button onClick={Auth.signOut}>Sign Out</button>
+						</span>
+						: <a href="/login">Sign In</a>
+					}
```

Comment:
I think it might be more clear to write this as 2 separate binary statements instead of a ternary statement.